### DISCUSSION TITLE
Elasticsearch support within RODA

### DISCUSSION THREAD

**Discussion by jojsf** [keeps/roda#2817] [Ideas]
Hi,

I would like to have support for elasticsearch within Roda. 
Is there any plans for that in the future?
Therese is at least a few government organisations looking into that.

Br/
Stefan

**Comment by luis100** [comment]
Hello, we are in conversations for the development of ElasticSearch support in RODA, to be (co-)sponsored by (unnamed) government organizations. If all goes as expected, a (possibly commercial) extension of RODA would add support for ElasticSearch until next fall. Apache Solr would be maintained as the default supported indexing system on the community version.

**Reply by jojsf** [reply]
Next fall as in 2024 or 2023?

**Reply by luis100** [reply]
October 2023

**Comment by luis100** [comment]
Just to let you know that the sponsored development this not go through. Generally, it was found to be too expensive for the initial development and following maintenance for the value it would provide, namely for institutions to use and maintain a indexing system they are already used to (Elastic Search) instead of getting to know a new one (Apache Solr, the currently supported platform).
